core-image.html

```diff
@@ -202,7 +202,7 @@
       if (this.preload) {
         if (this.fade) {
           if (!this._placeholderEl) {
-            this._placeholderEl = this.shadowRoot.querySelector('#placeholder');
+            this._placeholderEl = this.shadowRoots["core-image"].querySelector('#placeholder');
```

Comment:
Or actually since it's inside another `<template>` you can do `<template id="placeholderTemplate">` and then

    this._placeholderEl = this.$.placeholderTemplate.querySelector('#placeholder');

"placeholderTemplate" might be too verbose, feel free to shorten.